documents/payment.md

```diff
@@ -630,6 +630,196 @@ Doc reference [doc](https://razorpay.com/docs/payments/payment-methods/cards/aut
 
 -------------------------------------------------------------------------------------------------------
 
+### Create Payment Json (Third party validation)
+
+```rb
+param_attr = {
+  "amount": "100",
+  "currency": "INR",
+  "email": "gaurav.kumar@example.com",
+  "contact": "9123456789",
+  "order_id": "order_JkVtS9XJdk2u0B",
+  "method": "netbanking",
+  "bank": "HDFC"
+}
+
+Razorpay::Payment.create_json_payment(param_attr)
+
+```
+
+**Parameters:**
+| Name        | Type    | Description                          |
+|-------------|---------|--------------------------------------|
+| amount*          | integer | Amount of the order to be paid  |
+| currency*   | string  | The currency of the payment (defaults to INR)                                  |
+| order_id*        | string  | The unique identifier of the order created. |
+| email*        | string      | Email of the customer                       |
+| contact*      | string      | Contact number of the customer              |
+| method*      | string  | Possible value is `netbanking` |
+| bank*      | string      | The customer's bank code.For example, `HDFC`.|
+
+ please refer this [doc](https://razorpay.com/docs/payments/third-party-validation/s2s-integration/netbanking#step-3-create-a-payment) for params
+
+**Response:** <br>
+```json
+{
+  "razorpay_payment_id": "pay_GAWOYqPlvrtPSi",
+  "next": [
+    {
+      "action": "redirect",
+      "url": "https://api.razorpay.com/v1/payments/pay_GAWOYqPlvrtPSi/authorize"
+    }
+  ]
+}
+```
+-------------------------------------------------------------------------------------------------------
+### Create Payment UPI s2s / VPA token (Third party validation)
+
+```rb
+param_attr = {
+  "amount": 200,
+  "currency": "INR",
+  "order_id": "order_GAWRjlWkVcRh0V",
+  "email": "gaurav.kumar@example.com",
+  "contact": "9123456789",
+  "method": "upi",
+  "customer_id": "cust_EIW4T2etiweBmG",
+  "save": 1,
+  "ip": "192.168.0.103",
+  "referer": "http",
+  "user_agent": "Mozilla/5.0",
+  "description": "Test flow",
+  "notes": {
+    "note_key": "value1"
+  },
+  "upi": {
+    "flow": "collect",
+    "vpa": "gauravkumar@exampleupi",
+    "expiry_time": 5
+  }
+}
+
+Razorpay::Payment.create_upi(para_attr)
```

Comment:
- Error on create_upi 
- Also change variable name from para_attr to param_attr

Reply:
try this code snippet with these order id's
`order_JryDO5A3pU0533, order_JryDcxfWtyWuQO, order_JryDlAaPomSXi0`

para_attr = {
  "amount": 100,
  "currency": "INR",
  "order_id": "order_JryClMTXJvYc3H",
  "email": "gaurav.kumar@example.com",
  "contact": "9123456789",
  "method": "upi",
  "customer_id": "cust_JkRs374KyewdSU",
  "save": 1,
  "ip": "192.168.0.103",
  "referer": "http",
  "user_agent": "Mozilla/5.0",
  "description": "Test flow",
  "notes": {
    "note_key": "value1"
  },
  "upi": {
    "flow": "collect",
    "vpa": "success@razorpay",
    "expiry_time": 5
  }
}

Razorpay::Payment.create_upi(para_attr)